src/components/Badges/_Badge.scss

```diff
@@ -0,0 +1,83 @@
+.coa-Badge {
+  align-items: center;
+  display: flex;
+  font-family: $coa-font-sans;
+  font-weight: $coa-font-semibold;
+  height: $coa-badge-height + $coa-badge-step-radius;
+  justify-content: center;
+  margin: $coa-badge-margin-tb $coa-badge-margin-lr;
+  overflow: hidden;
+  position: relative;
+  width: $coa-badge-width;
+
+  @include media($coa-medium-screen) {
+    height: $coa-badgelarge-height + $coa-badgelarge-step-radius;
+    margin: $coa-badgelarge-margin-tb $coa-badgelarge-margin-lr;
+    width: $coa-badgelarge-width;
+  }
+}
+
+.coa-Badge__background {
+  bottom: 0;
+  height: $coa-badge-height;
+  position: absolute;
+  width: $coa-badge-width;
+  path {
+    fill: transparent;
+    stroke: $coa-color-gray-light;
+    stroke-width: $coa-badge-stroke;
+  }
+  polygon {
+    fill: $coa-color-green-light;
+  }
+
+  @include media($coa-medium-screen) {
+    height: $coa-badgelarge-height;
+    width: $coa-badgelarge-width;
+  }
+}
+
+.coa-Badge__title {
+  font-size: $coa-font-size-xsmall;
+  line-height: $coa-line-height-xsmall;
+  padding: 1rem;
+  text-align: center;
+  @include media($coa-medium-screen) {
+    font-size: $coa-font-size-md-small;
+    line-height: $coa-line-height-small;
+    padding: 2rem;
+  }
+}
+
+.coa-Badge__step {
+  align-items: center;
+  background: $coa-color-white;
+  border: ($coa-badge-stroke - 1) solid $coa-color-purple-dark;
+  border-radius: 50%;
+  display: flex;
+  font-size: $coa-font-size-md-small;
+  font-weight: $coa-font-bold;
+  height: $coa-badge-step-radius * 2;
+  justify-content: center;
+  position: absolute;
+  top: 0;
+  width: $coa-badge-step-radius * 2;
+  @include media($coa-medium-screen) {
+    font-size: $coa-font-size-medium;
+    height: $coa-badgelarge-step-radius * 2;
+    width: $coa-badgelarge-step-radius * 2;
+  }
+}
+
+.coa-Badge--isActive {
+  .coa-Badge__background {
+    path {
+      stroke: $coa-color-purple-dark;
+      stroke-width: $coa-badge-stroke + 2px;
+    }
+  }
+  .coa-Badge__title {
+    text-decoration-color: $coa-color-green-light; /* limited browser compat */
```

Comment:
what does this comment mean?

Reply:
text decoration color doesn't really have broad browser compatibility so changing the underline color isn't simple. so we shouldn't expect the underline to be green in most cases. Open to rewording to make this comment more clear if you have suggestions.